vmm/sandbox/src/stratovirt/mod.rs

```diff
@@ -69,29 +70,22 @@ pub(crate) const STRATOVIRT_START_TIMEOUT_IN_SEC: u64 = 10;
 // but skip all the fields serialization.
 #[derive(Default, Serialize, Deserialize)]
 pub struct StratoVirtVM {
-    #[serde(skip)]
     id: String,
-    #[serde(skip)]
     config: StratoVirtConfig,
     #[serde(skip)]
     devices: Vec<Box<dyn StratoVirtDevice + Sync + Send>>,
     #[serde(skip)]
     hot_attached_devices: Vec<Box<dyn StratoVirtHotAttachable + Sync + Send>>,
```

Comment:
the hot_attached_devices for qemu should be recovered when restart, otherwise the devices of the container will be residual after the container is restarted.

Reply:
But I found that the hot_detach is not implemented for stratovirt, how to detach the devices from vm after a container stopped?

Reply:
Yes, stratovirt vm hot_detach is not implemented yet, which may cause residual problems as you say, especially when using dm as storage driver. But as lack of device recovery has no influence on basic sandbox and container management on overlay scene, and it is a little complicated, is it ok if I open a new issue to track device recovery, not implement in this commit?